### ISSUE TITLE
Incorrect logic, "typeable" mode

### ISSUE THREAD

**Issue by koss-shtukert** [charliekassel/vuejs-datepicker#573]
Please see my code on codesandbox:
[https://codesandbox.io/s/14qx8j0774](https://codesandbox.io/s/14qx8j0774)

Problem:
 have disabled all dates before the current date. 
When input previous date i see that in the calendar current date is selected (because i validate it in "selected" method: `if (moment(value, 'DDMMMYYYY').isBefore(moment(), 'day'))`), but in input i still see my input date.

May be the calendar must control the logic, when I fill in "incorrect" date via keyboard input, the calendar must replace it with the current date or the "open-date" as default value

**Comment by charliekassel** [comment]
This looks good to me.
I'm seeing in your code: ```disabledDates: {
            to: moment().subtract(1, 'day').toDate()
        }```
Which is disabling everything up to yesterday.
Today is the 7th August and everything is disabled to the 6th.

**Comment by koss-shtukert** [comment]
Yes, but i can type on the keyboard 6th August, and in input field it will be present as "06Aug2018" and it's wrong because i disable everything up to today (7th August)

**Comment by charliekassel** [comment]
Got it. OK reopening.

**Comment by s695** [comment]
Maybe this could be merged and it can be the solution for this bug ?
https://github.com/charliekassel/vuejs-datepicker/pull/718/commits/486c69fa43bbad6fe76b893475d4d031d1ff3084
